[3.7] docs(calendar, daterangepicker): size and orientation documentation (#1188)
telerik/blazor-docs: components/calendar/appearance.md

---
title: Appearance
page_title: Calendar Appearance
description: Appearance settings of the Calendar for Blazor.
slug: calendar-appearance
tags: telerik,blazor,calendar,appearance
published: True
position: 30
---

# Appearance Settings

> You can control the appearance of the Calendar by setting the following attribute:

Reviewer: Try to make the first sentence sound a bit more introductory and detached from the article structure. This will make it future-proof as well.

```suggestion
This article outlines the available Calendar parameters, which control its appearance.
```

> ## Size

Reviewer: It's good to explain what does this parameter change and what it doesn't. Some customers have incorrect expectations.

You can increase or decrease the size of the Calendar by setting the `Size` attribute to a member of the `Telerik.Blazor.ThemeConstants.Calendar.Size` class:

> | Class members | Manual declarations |

Reviewer: ```suggestion
| Class member | Manual declaration |
```

> |------------|--------|

Reviewer: We are not aiming for ASCII-art style, so better use the minimum number of hyphens.

```suggestion
|---|---|
```
